### ISSUE TITLE
setPreference not working any more with newest version

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#12]
```
Operating system : Windows 7, 64bit
.Net Framework version : 4.03
Office Version : 10
SeleniumWrapper version : 1.0.7.0

What steps will reproduce the problem?

Dim selenium As New SeleniumWrapper.WebDriver  
selenium.setPreference "browser.download.folderList", 2

What is the expected output? What do you see instead?

Selenium should start Firefox. Instead I get error 438. "Object doesn't support 
this property or method."

It worked with the 1.0.6 builds, but stopped with 1.0.7.


```

Original issue reported on code.google.com by `gruberal...@gmail.com` on 17 Dec 2012 at 9:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 17 Dec 2012 at 8:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 1.0.8.0
Could you tell me if you still get the error?

For the record, i made the assembly version a constant as it was linked to the 
workbook and as Excel was trying to use a previous version of the driver.
```

Original comment by `florentbr` on 6 Jan 2013 at 12:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for fixing! Everything is working now.
```

Original comment by `gruberal...@gmail.com` on 7 Jan 2013 at 9:02
